pkgs/development/python-modules/zstd/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchgit, fetchpatch }:
+
+buildPythonPackage rec {
+  pname = "zstd";
+  version = "1.3.5.1";
+
+  src = fetchgit {
```

Comment:
I can't remember, were you using `fetchPypi` initially? If you want to fetch from github, that's fine probably, but then, please, use the preferred `fetchFromGitHub` instead of `fetchgit`.